packages/cfd/src/Components/cfd-poi.tsx

```diff
@@ -24,28 +22,50 @@ export type TCFDPOIProps = {
     index: number;
     onSubmit: (index: number, value: TCFDValue) => void;
     value: TCFDValue;
-    account_status?: GetAccountStatus;
-    addNotificationByKey: (key: string) => void;
-    fetchResidenceList?: () => void;
+    addNotificationMessageByKey: (key: string) => void;
```

Comment:
```suggestion
    addNotificationMessageByKey: (key: TCFDNotificationByKey) => void;
```

Reply:
@hamza-deriv type `string` and `{key: string}` are two different types